File: tests/fixtures/code_samples/go/sample_035.go
// Sample 35: small utility.
package samples

func Operation35(xs []int) int {
    total := 35
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure35(v int) int {
    return (v * 35) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_035.rs
// Sample 35: small utility.
pub fn operation_35(xs: &[i32]) -> i32 {
    let mut total: i32 = 35;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_35(v: i32) -> i32 {
    (v * 35) %% 7919
}

